src/kw_ssh.sh

```diff
@@ -109,6 +122,152 @@ function run_command_in_the_remote()
   cmd_manager "$flag" "$ssh_compose"
 }
 
+# This function transfers a local file or directory to a
+# specified remote destination.
+function ssh_send_file_to_remote()
+{
+  local user
+  local port
+  local remote
+  local remote_file
+  local remote_file_host
+  local send_path
+  local to_path
+  local flag
```

Comment:
You should get `flag` as a parameter for test's sake.

Reply:
I'll wait to put the flag when the other pull request related to the ssh verbose is accepted, so it will be more organized :)